isic_archive/models/image.py

```diff
@@ -68,6 +78,29 @@ def createEmptyImage(self, originalFileRelpath, parentFolder, creator, dataset,
 
         return image
 
+    def resetImageForIngest(self, image):
+        originalFile = self.originalFile(image)
```

Comment:
I'm concerned about the fact that `Image.originalFile` claims it will sometimes return the wrong file for old images with no large_image.

Since we can't rely on large_image to always provide the original file, I think we should move towards being more explicit in how we identify each type of image file.

We could either set:
1. `image['originalFileId'] = originalFile['_id']`
2. `file['imageRole'] = 'original'`

and so on for `superpixel`, `large_image`, etc. as a much more direct and reliable way to reference these files.

I think the major difference in the approaches is that 2. theoretically allows for multiple files in the same role. Although my first instinct is to prevent this with 1., I realize that it doesn't provide a truly strict constraint, as files could always leak if a `originalFileId` field were re-set without deleting the old file. The advantage of 2. is that it never leaks files, and it'd be much simpler to do a health check by asserting that each image has exactly one file of each role.

This whole thing is another side-task, but perhaps its worth adding now, to give more sanity and assurance to any maintenance going forward.